OracleUnifiedDirectorySM/README.md

```diff
@@ -0,0 +1,198 @@
+Oracle Unified Directory Service Manager (OUDSM) on Docker
+==========================================================
+
+## Contents
+
+1. [Introduction](#1-introduction-1)
+2. [Hardware and Software Requirements](#2-hardware-and-software-requirements)
+3. [Prerequisites](#3-prerequisites)
+4. [Building OUDSM Docker Image](#4-loading-or-building-oudsm-docker-image)
+5. [Preparing to Run OUDSM Docker Image](#5-preparing-to-run-oudsm-docker-image)
+6. [Running OUDSM Docker Container](#6-running-oudsm-docker-container)
+
+# 1. Introduction
+This project offers Dockerfile and scripts to build an Oracle Unified Directory Service Manager image based on 12cPS4 (12.2.1.4.0) release. Use this Docker Image to facilitate installation, configuration, and environment setup for DevOps users. 
+
+This Image refers to binaries for OUD Release 12.2.1.4.0 and it has the capability to create FMW Infrastructure domain with OUDSM application deployed in container with OUDSM deployed which can be targeted for development and testing.
+
+***Image***: oracle/oudsm:12.2.1.4.0
+
+# 2. Hardware and Software Requirements
+Oracle Unified Directory Service Managedr Docker Image has been tested and is known to run on following hardware and software:
+
+## 2.1 Hardware Requirements
+
+| Hardware  | Size  |
+| :-------: | :---: |
+| RAM       | 16GB  |
+| Disk Space| 200GB+|
+
+## 2.2 Software Requirements
+
+|       | Version                        | Command to verify version |
+| :---: | :----------------------------: | :-----------------------: |
+| OS    | Oracle Linux 7.3 or higher     | more /etc/oracle-release  |
+| Docker| Docker version 18.03 or higher | docker version            |
+
+# 3. Prerequisites
+
+## 3.1 Pulling the Oracle JDK (Server JRE) base image
```

Comment:
In the repo, we always recommend building images from Dockerfiles. You should link to the documentation on how to build the `serverjre:8` image as well as how to pull it from OCR.

Reply:
We would like to recommend customers to use imagetool only for building docker images. Updated readme files are committed with reference to building images using imagetool.

Reply:
> OUDSM is a product related to OUD but OUD and OUDSM both are separate products. OUD is a LDAP Server built using Java and OUDSM is web console application deployed on Weblogic. Considering them as separate products and separate docker images, we have kept dedicated top level folder for each.

They're not separate groups though. OUDSM is a web console for OUD, so it should sit underneath the `OracleUnifiedDirectory` folder. This reduces the overall amount of top-level products.

Reply:
Considering them as separate docker images built on top of different base (JRE and FMW-Infrastructure), we would like to keep them at different top-level product directories.

Reply:
It's not about seperate images or different bases. It's all part of Oracle Unified DIrectory. It makes no organisational sense to have them as two top-level directories.

Reply:
Added files for building image using Dockerfile